src/iminuit/_minuit.py

```diff
@@ -1452,9 +1454,9 @@ def __eq__(self, other):
         return len(self) == len(other) and all(x == y for x, y in zip(self, other))
 
     def __repr__(self):
-        s = "<%s of Minuit at %x>" % (self.__class__.__name__, id(self._minuit))
+        s = "<{} of Minuit at {:x}>".format(self.__class__.__name__, id(self._minuit))
```

Comment:
Very nice that it can replace old-style formatting - I wonder if it has a 2.7 mode I can apply to very old projects? :)